src/server/services/dataService/queries/changefeedForMemberPhaseChanged.js

```diff
@@ -0,0 +1,8 @@
+import r from '../r'
+
+export default function changefeedForMemberPhaseChanged() {
+  return r.table('members').changes()
+    .filter(
+      r.row('old_val')('phaseId').ne(r.row('new_val')('phaseId')) // phase id changes
```

Comment:
This wouldn't trigger an event in the case that the old `phaseId` is undefined. Should be:
```js
  r.row('old_val')('phaseId').default(null)
    .ne(
      r.row('new_val')('phaseId').default(null)
    )
```